Readme.md

```diff
@@ -7,28 +7,46 @@
 # Tracee: Runtime Security and Forensics using eBPF
 
 Tracee uses eBPF technology to tap into your system and give you access to hundreds of events that help you understand how your system behaves.
-In addition to basic observability events about system activity, Tracee adds a collection of sophisticated security events that expose more advanced behavioral patterns. You can also easily add your own events using the popular [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/) language.
+In addition to basic observability events about system activity, Tracee adds a collection of sophisticated security events that expose more advanced behavioral patterns. 
 Tracee provides a rich filtering mechanism that allows you to eliminate noise and focus on specific workloads that matter most to you.
 
-To learn more about Tracee, check out the [documentation](https://aquasecurity.github.io/tracee).
+**Key Features:**
+* Kubernetes native installation
+* Hundreds of default events
+* Up-to-date tracking of malicious behaviour through signatures provided by the Aqua Security Research Team 
+* Easy configuration through Tracee Policies 
+* Accessible user experience for cluster administrators
 
+> We release new features and changes on a monthly basis. Learn more about the letest release in our [discussions.](https://github.com/aquasecurity/tracee/discussions)
+
+**Integrations**
+
+* Rego: You can easily define custom events that you would like Tracee to track using the popular [Rego](https://www.openpolicyagent.org/docs/latest/policy-language/) language.
+* Monitoring: Additionally, we have several tutorials that showcase integration between monitoring solutions and Tracee.
 ## Quickstart
 
-You can easily start experimenting with Tracee using the Docker image as follows:
+Either:
+* [Install Tracee in your Kubernetes cluster.](https://aquasecurity.github.io/tracee/latest/getting-started/kubernetes-quickstart)
+* [Experiment using the Tracee container image.](https://aquasecurity.github.io/tracee/v0.17/#quickstart)
```

Comment:
the docs main page should be similar to the readme (like before)

Reply:
that's something we can change once we agree on the README change?

Reply:
ok so let's not resolve this comment

Reply:
edited the main index page to reflect the README